manuscript/recipes/photoprism.md

```diff
@@ -0,0 +1,176 @@
+hero: Your own private google photos
+
+# Photoprism
+
+[Photoprism™](https://github.com/photoprism/photoprism) *" is a server-based application for browsing, organizing and sharing your personal photo collection. It makes use of the latest technologies to automatically tag and find pictures without getting in your way. Say goodbye to solutions that force you to upload your visual memories to the cloud."*
+
+
+![Photoprism Screenshot](../images/photoprism.png)
+
+## Ingredients
+
+1. [Docker swarm cluster](/ha-docker-swarm/design/) with [persistent shared storage](/ha-docker-swarm/shared-storage-ceph.md)
+2. [Traefik](/ha-docker-swarm/traefik_public) configured per design
+3. DNS entry for the hostname you intend to use, pointed to your [keepalived](ha-docker-swarm/keepalived/) IP
+
+## Preparation
+
+### Setup data locations
+
+First we need a folder to map the photoprism config file:
+```
+mkdir /var/data/photoprism/config
+```
+
+We will need a location to store photoprism thumbnails, as they can be recreated anytime (althought depending on your colleciton size it could take a while), we store them on a "non-backed" folder
```

Comment:
Text in parentheses should be italicised, (_like this_)

Reply:
Also, spelling issues above on "colleciton" and "non-backed" (_this should be "non-backed-up"_)